tests/longevity-48h-verify-multiDC.yaml

```diff
@@ -1,27 +1,24 @@
 test_duration: 2860
 prepare_write_cmd:  "cassandra-stress write cl=ALL n=200200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..200200300"
 prepare_verify_cmd: "cassandra-stress read  cl=ALL n=200200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..200200300"]
-stress_cmd: ["cassandra-stress write cl=QUORUM duration=2860m -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=00200300..600200300"]
-stress_read_cmd: ["cassandra-stress read cl=ONE duration=100m -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..200200300"]
-cassandra_stress_duration: 10080
-cassandra_stress_threads: 1000
-cassandra_stress_population_size: 100000000000
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=2860m -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=400200300..600200300"]
+stress_read_cmd: ["cassandra-stress read cl=ONE duration=2860m -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=3,us-west-2scylla_node_west=3)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=1..200200300"]
 n_db_nodes: '3 3'
 n_loaders: 1
 n_monitor_nodes: 1
 nemesis_class_name: 'LimitedChaosMonkey'
 nemesis_interval: 5
-user_prefix: 'sct-multidc'
+user_prefix: longevity-multidc-200gb-vfy-VERSION
 failure_post_behavior: keep
 space_node_threshold: 6442
 ip_ssh_connections: 'public'
 
-ami_id_db_scylla_desc: '2-3-0'
+ami_id_db_scylla_desc: VERSION
 
 use_mgmt: true
 mgmt_port: 10090
-scylla_repo_m: 'http://repositories.scylladb.com/scylla/repo/7b02fff5-e4d0-4e4d-ad12-e605ca4873c2/centos/scylladb-2018.1.repo'
-scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/branch-1.2/44/scylla-manager.repo'
+scylla_repo_m: SCYLLA_ENTERPRISE_REPO
+scylla_mgmt_repo: SCYLLA_MGR_REPO
 
 
 backends: !mux
```

Comment:
Replace all ami-id with AMI_ID_EAST and AMI_ID_WEST

Reply:
done